SLHCUpgradeSimulations/Configuration/python/combinedCustoms.py

```diff
@@ -10,34 +12,109 @@
+#     process=fixRPCConditions(process)
+#     process=fixDTAlignmentConditions(process)
+#     process=digiOnlyMuonDetectors(process)
+#     process=recoOnlyMuonDetectors(process)
+#     return process
+
+
 def noCrossing(process):
     process=customise_NoCrossing(process)
     return process
 
 
 def fixRPCConditions(process):
+    process.simMuonRPCDigis.digiModel = cms.string('RPCSimAverageNoiseEffCls')
     if not hasattr(process.GlobalTag,'toGet'):
         process.GlobalTag.toGet=cms.VPSet()
     process.GlobalTag.toGet.extend( cms.VPSet(
         cms.PSet(record = cms.string("RPCStripNoisesRcd"),
-                 tag = cms.string("RPCStripNoise_upscope_mc_v2"),
+                 tag = cms.string("RPC_testCondition_192Strips_mc"),
                  connect = cms.untracked.string("frontier://FrontierProd/CMS_COND_31X_RPC")
+                 ),
+        cms.PSet(record = cms.string("RPCClusterSizeRcd"),
+                 tag = cms.string("RPCClusterSize_PhaseII_mc"),
+                 connect = cms.untracked.string("frontier://FrontierProd/CMS_COND_36X_RPC")
                  )
         )
-                                    )
+    )
     return process
 
+def fixDTAlignmentConditions(process):
+    if not hasattr(process.GlobalTag,'toGet'):
+        process.GlobalTag.toGet=cms.VPSet()
+    process.GlobalTag.toGet.extend( cms.VPSet(
+            cms.PSet(record = cms.string("DTAlignmentErrorExtendedRcd"),
+                     tag = cms.string("MuonDTAPEObjectsExtended_v0_mc"),
+                     connect = cms.untracked.string("frontier://FrontierProd/CMS_COND_ALIGN_000")
+                 )
+            )
+    ),
+    process.GlobalTag.toGet.extend( cms.VPSet(
+            cms.PSet(record = cms.string("DTRecoUncertaintiesRcd"),
+                     tag = cms.string("DTRecoUncertainties_True_v0"),
+                     connect = cms.untracked.string("frontier://FrontierProd/CMS_COND_DT_000")
+                 )
+            )
+    ),
+    return process
+
+def fixCSCAlignmentConditions(process):
+    if not hasattr(process.GlobalTag,'toGet'):
+        process.GlobalTag.toGet=cms.VPSet()
+    process.GlobalTag.toGet.extend( cms.VPSet(
+            cms.PSet(record = cms.string("CSCAlignmentErrorExtendedRcd"),
+                     tag = cms.string("MuonCSCAPEObjectsExtended_v0_mc"),
+                     connect = cms.untracked.string("frontier://FrontierProd/CMS_COND_ALIGN_000")
+                 )
+            )
+    ),
+#     process.GlobalTag.toGet.extend( cms.VPSet(
+#             cms.PSet(record = cms.string("DTRecoUncertaintiesRcd"),
+#                      tag = cms.string("DTRecoUncertainties_True_v0"),
+#                      connect = cms.untracked.string("frontier://FrontierProd/CMS_COND_DT_000")
+#                  )
+#             )
+#     ),
+    return process
+
+
+# --> delete
```

Comment:
here and above: what's the purpose of commented out code?
If not needed, please remove.